core/src/main/java/com/yahoo/oak/Header.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2020, Verizon Media.
+ * Licensed under the terms of the Apache 2.0 license.
+ * Please see LICENSE file in the project root for terms.
+ */
+
+package com.yahoo.oak;
+
+import sun.misc.Unsafe;
+
+class Header {
+
+    /*
+    * Long Header: int version + int lock
+    * 0...  ...31 | 32...                  ...61| 62 63
+    *  version    |   lock: current_readers#    | lock state
+    *
+    * lock state: 0x00 - FREE, 0x01 - LOCKED, 0x10 - DELETED, 0x11 - MOVED
+    *
+    * The length (in integer size) of the data is also held just following the header (in long size)
+    * The length is set once upon header allocation and later can only be read. */
+
+    enum LockStates {
+        FREE(0), LOCKED(1), DELETED(2), MOVED(3);
+
+        public final int value;
+
+        LockStates(int value) {
+            this.value = value;
+        }
+    }
+
+    private static final int VERSION_SIZE = 4;
+    private static final int VERSION_OFFSET = 0;
+
+    private static final int LOCK_STATE_MASK = 0x3;
+    private static final int LOCK_STATE_SHIFT = 2;
```

Comment:
`LOCK_STATE_MASK` and `LOCK_STATE_SHIFT` are unused.

Reply:
removed